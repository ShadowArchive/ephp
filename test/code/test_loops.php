<?php

$primes = [];
for ($i=0; $i<100; $i++) {
    $primes[] = false;
}

$primes[1] = true;
$primes[2] = true;

for ($i=3; $i<100; $i++) {
    $is_prime = true;
    for ($j=2; $j<$i; $j++) {
        if ($i % $j == 0) {
            $is_prime = false;
            break;
        }
    }
    $primes[$i] = $is_prime;
}

print_r($primes);

