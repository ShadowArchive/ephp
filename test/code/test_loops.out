Array
(
    [0] => 
    [1] => 1
    [2] => 1
    [3] => 1
    [4] => 
    [5] => 1
    [6] => 
    [7] => 1
    [8] => 
    [9] => 
    [10] => 
    [11] => 1
    [12] => 
    [13] => 1
    [14] => 
    [15] => 
    [16] => 
    [17] => 1
    [18] => 
    [19] => 1
    [20] => 
    [21] => 
    [22] => 
    [23] => 1
    [24] => 
    [25] => 
    [26] => 
    [27] => 
    [28] => 
    [29] => 1
    [30] => 
    [31] => 1
    [32] => 
    [33] => 
    [34] => 
    [35] => 
    [36] => 
    [37] => 1
    [38] => 
    [39] => 
    [40] => 
    [41] => 1
    [42] => 
    [43] => 1
    [44] => 
    [45] => 
    [46] => 
    [47] => 1
    [48] => 
    [49] => 
    [50] => 
    [51] => 
    [52] => 
    [53] => 1
    [54] => 
    [55] => 
    [56] => 
    [57] => 
    [58] => 
    [59] => 1
    [60] => 
    [61] => 1
    [62] => 
    [63] => 
    [64] => 
    [65] => 
    [66] => 
    [67] => 1
    [68] => 
    [69] => 
    [70] => 
    [71] => 1
    [72] => 
    [73] => 1
    [74] => 
    [75] => 
    [76] => 
    [77] => 
    [78] => 
    [79] => 1
    [80] => 
    [81] => 
    [82] => 
    [83] => 1
    [84] => 
    [85] => 
    [86] => 
    [87] => 
    [88] => 
    [89] => 1
    [90] => 
    [91] => 
    [92] => 
    [93] => 
    [94] => 
    [95] => 
    [96] => 
    [97] => 1
    [98] => 
    [99] => 
)
